report/src/main/java/com/megaease/easeagent/report/encoder/log/LogWriter.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2022, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.megaease.easeagent.report.encoder.log;
+
+import com.megaease.easeagent.plugin.api.logging.AccessLogInfo;
+import zipkin2.internal.JsonEscaper;
+import zipkin2.internal.WriteBuffer;
+
+import java.util.Map;
+
+public class LogWriter implements WriteBuffer.Writer<AccessLogInfo> {
+    static final String TYPE_FIELD_NAME = "\"type\":\"";
+    static final String TRACE_ID_FIELD_NAME = ",\"trace_id\":\"";
+    static final String SPAN_ID_FIELD_NAME = ",\"span_id\":\"";
+    static final String PARENT_ID_FIELD_NAME = ",\"pspan_id\":\"";
+    static final String SERVICE_FIELD_NAME = ",\"service\":\"";
+    static final String SYSTEM_FIELD_NAME = ",\"system\":\"";
+    static final String CLIENT_IP_FIELD_NAME = ",\"client_ip\":\"";
+    static final String USER_FIELD_NAME = ",\"user\":\"";
+    static final String RESPONSE_SIZE_FIELD_NAME = ",\"response_size\":\"";
+    static final String REQUEST_TIME_FIELD_NAME = ",\"request_time\":\"";
+    static final String CPU_ELAPSED_TIME_FIELD_NAME = ",\"cpuElapsedTime\":\"";
+    static final String URL_FIELD_NAME = ",\"url\":\"";
+    static final String METHOD_FIELD_NAME = ",\"method\":\"";
+    static final String STATUS_CODE_FIELD_NAME = ",\"status_code\":\"";
+    static final String HOST_NAME_FIELD_NAME = ",\"host_name\":\"";
+    static final String HOST_IPV4_FIELD_NAME = ",\"host_ipv4\":\"";
+    static final String CATEGORY_FIELD_NAME = ",\"category\":\"";
+    static final String MATCH_URL_FIELD_NAME = ",\"match_url\":\"";
+    static final String TIMESTAMP_FIELD_NAME = ",\"timestamp\":\"";
+    static final String HEADERS_FIELD_NAME = ",\"headers\":{";
+    static final String QUERIES_FIELD_NAME = ",\"queries\":{";
```

Comment:
Are you sure you want to enumerate all fields to serialize a json?

Reply:
borrow from Span(zipkin-brave) serialising implementation, this implementation shows better performance than Jackson `ObjectMapper` API in performance test.